### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by nomurakeiya** [lartpang/HDFNet#5]
I am very interested in your proposal and would love to use it.
However, since I cannot create a Baidu account, can you distribute the trained model with other cloud services?

Thank you for your work

**Comment by lartpang** [comment]
📆  I will update the pretrained parameter files to GitHub. 😸

**Comment by nomurakeiya** [comment]
Thank you!
I look forward to your upload.

**Comment by lartpang** [comment]
@nomurakeiya https://github.com/lartpang/HDFNet/releases/tag/v0.1

**Comment by nomurakeiya** [comment]
@lartpang 
Thank you for your wonderful work! !!

**Comment by lartpang** [comment]
Besides, I have not confirmed whether the source code packaged here is the latest version. It is best to clone the latest code in the repository directly.

---

The structural parameters and training settings of the current model may not be optimal, you can try it yourself.

**Comment by nomurakeiya** [comment]
@lartpang 

Is it possible to detect the number of objects (people, etc.) by entering only the image?
Thank you for your advice.

**Comment by lartpang** [comment]
@nomurakeiya 

This problem is suitable for saliency instance segmentation methods to deal with. 

For salient object detection methods like HDFNet, traditional computer vision methods can be used to count objects efficiently only if they do not overlap.

**Comment by nomurakeiya** [comment]
@lartpang 
oh, I see...
So HDFNet isn't suitable for counting crowds, right?

by the way,Is it possible to obtain the detection result?
I don't care about accuracy.
I don't understand the command.

**Comment by lartpang** [comment]
@nomurakeiya 
It is not a method designed for counting. 
Its output is a probability map indicating the probability that each pixel belongs to the salient object.
As I mentioned above, if there is no overlap between object regions, then the connectivity of regions can be used to distinguish different objects and calculate the number of objects.
- https://en.wikipedia.org/wiki/Connected-component_labeling

Besides, I'm not sure what "the detection result" refers to. Is it the prediction result on the RGBD SOD task?
